package-lock.json

```diff
@@ -10,8 +10,8 @@
       "dependencies": {
         "@formatjs/intl-pluralrules": "4.0.6",
         "@formatjs/intl-relativetimeformat": "6.2.3",
```

Comment:
are those package-lock json changes intended?

Reply:
More or less. They're not an intentional part of this change. It was a natural result of a prior commit/PR bumping deps without updating the lock file (https://github.com/mattermost/mattermost-webapp/commit/3f79e3b270982d7e74889da84d1646bfc7c39e90) . At this point, all users running `make run` will trigger the same lockfile update, so figured it makes the most sense to go ahead and do it in this PR. Happy to revert and let someone else make the change if you think that would be better.